Code-the-Dream-School/MyKitchenApp-front
# Google login dev

## Description

Adds the ability for a user to choose one-click google sign in
## Resouce links or research articles

## Related Issue


closes #88 links #18
## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before




### After

<img width="550" alt="image" src="https://user-images.githubusercontent.com/88516807/200772275-894a21f7-d58b-4eae-8f67-767d0ab0177b.png">

## Testing Steps / QA Criteria


log in using google. if your token is already saved, open dev tools, open application and delete the token there and go back to localhost:3000 (or wherever is opened)/signin and you will see the google button
